mozci/console/commands/decision.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+import os
+from datetime import datetime
+
+import taskcluster
+from cleo import Command
+
+from mozci import data
+from mozci.push import Push
+from mozci.util.memoize import memoized_property
+from mozci.util.taskcluster import get_proxy_queue
+
+
+class DecisionCommand(Command):
+    """
+    Taskcluster decision task to generate classification tasks
+
+    decision
+        {branch=mozilla-central : Branch the push belongs to (e.g autoland, try, etc).}
+        {--dry-run : Do not create tasks on taskcluster, simply output actions.}
+    """
+
+    def handle(self):
+        branch = self.argument("branch")
+        dry_run = self.option("dry-run")
+
+        self.queue = not dry_run and get_proxy_queue() or None
+
+        self.line(f"Process pushes from {branch}")
+
+        # List pushes
+        pushes = data.handler.get(
+            "pushes",
+            branch=branch,
+        )
+
+        for push in pushes:
+
+            # TODO: detect if a classification is already available
+            # and can be skipped
```

Comment:
Given our discussion from yesterday, we might not want to do this to avoid having to store state somewhere and instead just rely on the age of a push.
Let's remove the TODO and maybe file it as an "investigate XXX" issue.